File: src/src/components/Orders/OrderInfo.py
import reflex as rx


def renderOrder(order: dict[int, str, int, int]) -> rx.Component:
    return (
        rx.box(
            rx.text(order["name"]),
            rx.text("₹ " + str(order["quantity"] * order["price"])),
            class_name="gap-8 flex justify-between font-medium text",
        ),
    )


def OrderInfo(orderList: list[dict[int, str, int, int]]) -> rx.Component:
    return rx.box(
        # rx.foreach(orderList, renderOrder),
        rx.divider(),
        rx.box(
            rx.text("Total:"),
            rx.text("₹ " + str(10000)),
            class_name="gap-8 flex justify-between font-medium text text-lg",
        ),
        class_name="OrderInfo w-full mx-2 flex flex-col justify-around",
    )

File: frontend/src/components/Orders/OrderInfo.js
import React from "react";

const OrderInfo = ({ orderList }) => {
  return (
    <div className="OrderInfo w-full mx-2 flex flex-col justify-around">
      {orderList.map(({ id, name, quantity, price }) => {
        return (
          <p className="gap-8 flex justify-between font-medium text" key={id}>
            <span>
              {name} x {quantity}
            </span>
            <span>₹ {quantity * price}</span>
          </p>
        );
      })}
      <hr />
      <p className="gap-8 flex justify-between font-medium text text-lg">
        <span>Total:</span>
        <span>₹ {69420}</span>
      </p>
    </div>
  );
};

export default OrderInfo;
